osu.Game/Skinning/LegacySongProgress.cs

```diff
@@ -19,11 +19,14 @@ public partial class LegacySongProgress : SongProgress
         public override bool HandleNonPositionalInput => false;
         public override bool HandlePositionalInput => false;
 
+        public LegacySongProgress()
+        {
+            AutoSizeAxes = Axes.Both;
+        }
```

Comment:
I see what the intent was here but this is a little dodgy...

Would add inline commentary about what this is doing. The hope is that as we transition to better per-case handling as discussed we can remove this and just handle more sanely as part of the component.